implementations/app-typescript/src/helpers/is_keyword.rs

```diff
@@ -0,0 +1,20 @@
+use handlebars::handlebars_helper;
+use serde_json::{Value};
+
+handlebars_helper!(is_keyword: |val: Value| {
+    let s = val.as_str().unwrap();
```

Comment:
From the client sync convo earlier, do using less unwrap also applicable here?

Reply:
I'm not sure whether we can return a `Result` from a `handlebars_helper!`, but for sure I can go through and try to improve error messages.

Reply:
There isn't much to be done, honestly. I'm going to mark this resolved for now.

The `unwrap` you commented on, and the others like it, already provide useful error messages.